DataConnectors/AWS-S3/README.md

```diff
@@ -0,0 +1,8 @@
+
+# AWS S3 Scripts
+The scripts under this directory can be used to configure AWS S3 connectors.
+These are optional scripts instead of set AWS configurations manually.
+
+# Script pre requirements 
+* PowerShell [Installation instructions](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-7.1)
```

Comment:
Please use https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-7.1 instead of https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-7.1.
We don't use local  in URLs.